### DISCUSSION TITLE
Enforce default and max. dynamic sku function timeout for Linux consumption

### DISCUSSION THREAD

**Discussion by brettsam** [Azure/Azure-Functions#1919] [Design discussions]
Discussion for #1915

**Comment by brettsam** [comment]
Only note from the review:
- Detection should change to use `function.invoke.latency` metrics.

**Comment by fabiocav** [comment]
@balag0 do you have an issue tracking the work that needs to be done? If possible, it would be good to have that done in sprint 107

**Reply by balag0** [reply]
@pragnagopa actually fixed this already - https://github.com/Azure/azure-functions-host/pull/7438

**Reply by fabiocav** [reply]
Great! thank you
